MapboxNavigation/PollyVoiceController.swift

```diff
@@ -122,17 +122,26 @@ public class PollyVoiceController: RouteVoiceController {
     }
     
     func callSuperSpeak(_ text: String, error: String) {
+        pollyTask?.cancel()
+        
+        guard let audioPlayer = audioPlayer else {
+            super.speak(fallbackText, error: error)
+            return
+        }
+        
+        guard !audioPlayer.isPlaying else { return }
```

Comment:
Should we stop the audio player if some error has caused us to get into this code path?

Reply:
This is happening in #669.